custom_config_data/besu.json

```diff
@@ -14,7 +14,14 @@
     "preMergeForkBlock": 0,
     "terminalTotalDifficulty": 0,
     "shanghaiTime": 1694884704,
-    "ethash": {}
+    "ethash": {},
+    "cancunTime": 2000000000,
```

Comment:
The repo says Cancun TBD. Was this meant to be here?

Reply:
right not required to be here